fhir-term/src/main/java/com/ibm/fhir/term/util/CodeSystemSupport.java

```diff
@@ -340,8 +351,8 @@ public static boolean hasConceptProperty(Concept concept, Code code) {
      *     true if the code system is case sensitive, false otherwise
      */
     public static boolean isCaseSensitive(CodeSystem codeSystem) {
-        if (codeSystem != null && codeSystem.getCaseSensitive() != null) {
-            return java.lang.Boolean.TRUE.equals(codeSystem.getCaseSensitive().getValue());
+        if (codeSystem != null) {
+            return Boolean.TRUE.equals(codeSystem.getCaseSensitive());
```

Comment:
I actually think the old way was is more resilient to extensions (though its probably super unlikely we'd ever see an extension on this element).  Was there an issue with it?

Reply:
Reverted to old way